_posts/0000-01-03-LibertyToad.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Why is my avatar is angry?
```

Comment:
If you flip the avatar upside down then its happy.

Reply:
HA